src/postgres/tree/PostgreSQLTableTreeItem.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as _ from 'underscore';
+import * as vscode from 'vscode';
+import { AzureTreeItem } from 'vscode-azureextensionui';
+import { getThemeAgnosticIconPath } from '../../constants';
+import { getDocumentTreeItemLabel } from '../../utils/vscodeUtils';
+import { IPostgreSQLTreeRoot } from './IPostgreSQLTreeRoot';
+import { PostgreSQLSchemaTreeItem } from './PostgreSQLSchemaTreeItem';
+
+export class IPostgresTable {
```

Comment:
Yeah we don't use classes for something as simple as this. Just use an interface and an object